InitNiKom/InitNiKom.c

```diff
@@ -28,7 +28,7 @@ void createDirOrDie(char *dirname) {
   BPTR lock;
   struct FileInfoBlock __aligned fib;
   
-  if(lock = Lock(dirname, ACCESS_READ)) {
+  if((lock = Lock(dirname, ACCESS_READ))) {
```

Comment:
To be able to tell intentional assignments from bugs like the one fixed commit 3f0646a. :-)
Those bugs really happen, and it's nice for the compiler to be able to catch them, and the extra parentheses is simply a way to tell the compiler "I really know what I'm doing" so false warnings do not hide the real ones.